src/remote-config/remote-config-api-client.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseRemoteConfigError } from './remote-config-utils';
+import { FirebaseApp } from '../firebase-app';
+import * as validator from '../utils/validator';
+
+/**
+ * Class that facilitates sending requests to the Firebase Remote Config backend API.
+ *
+ * @private
+ */
+export class RemoteConfigApiClient {
```

Comment:
Indentation is not correct in this file. We intend with 2 spaces.

Reply:
Fixed